common-utils/network/test-fixture-handler.js

```diff
@@ -88,4 +88,26 @@ export class TestFixtureHandler {
       })
     })
   }
+
+  /**
+   * Read docs from a file and import them to OpenSearch only when the indices do not already exist
+   * in order to optimize the data uploading time
+   * @param {string, array} index An index or a set of indices 
+   * @param {string} indexMappingPath File path for mapping data
+   * @param {string} indexDataPath File path for the actual data
+   */
+  importJSONDocIfNeeded(index, indexMappingPath, indexDataPath) {
+    let queryString = '';
+    if (typeof index === 'string') {
+      queryString = index;
+    } else {
+      index.forEach(value => queryString += `${value},`);
+    }
```

Comment:
```suggestion
    if (typeof index === 'string') {
      queryString = index;
    } else if (Array.isArray(index)) {
      index.forEach(value => queryString += `${value},`);
    } else {
      return;
    }
```

Reply:
OR
```suggestion
if (typeof index === 'string') {
      queryString = index;
    } else if (Array.isArray(index)) {
      queryString += index.join(',') + ',';
    } else {
      return;
    }
```

Reply:
Donno if you want to have that tailing `,`, or what you wanna do if it is not an array or a string.